docs/source/markdown/podman-build.1.md

```diff
@@ -752,3 +752,6 @@ podman(1), buildah(1), containers-registries.conf(5), crun(8), runc(8), useradd(
 May 2018, Minor revisions added by Joe Doss <joe@solidadmin.com>
 
 December 2017, Originally compiled by Tom Sweeney <tsweeney@redhat.com>
+
+## FOOTNOTES
+<a name="Footnote1">1</a>: The Podman project is committed to inclusivity, a core value of open source. The `master` and `slave` mount propagation terminology used here is problematic and divisive, and should be changed. However, these terms are used in the Linux kernel and must be used as-is. When the kernel maintainers rectify this usage, Podman will suit immediately.
```

Comment:
Last line make no sense to me?

Podman will follow suit immediately
Or 
Podman will update to use the new kernel terminology.